src/pthread-main.js

```diff
@@ -0,0 +1,96 @@
+// Pthread Web Worker startup routine:
```

Comment:
Hmm, though the comments in the beginning of this file should make it clearer? Or do you have a rename suggestion that you'd prefer?